server/views/pages/incidentContact.njk

```diff
@@ -92,6 +92,33 @@
         value: formValues.incidentAvailability
       }) }}
 
+           {% from "govuk/components/select/macro.njk" import govukSelect %}
```

Comment:
This will need code changes for it to be pulled in 👍

Reply:
I have moved this to the top of the page, please confirm you're happy and it is correct. Thanks

Reply:
Sorry I meant the comment will be presented but you'd need changes in the code for it to pull the value out and do anything with it 🙂